src/modules/75-cd/components/PipelineSteps/CloudFormation/CloudFormationCreateStack/CloudFormationCreateStackRef.tsx

```diff
@@ -0,0 +1,650 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import cx from 'classnames'
+import * as Yup from 'yup'
+import { FieldArray } from 'formik'
+import {
+  Accordion,
+  Formik,
+  FormInput,
+  getMultiTypeFromValue,
+  MultiTypeInputType,
+  Color,
+  Layout,
+  Label,
+  MultiSelectOption,
+  Text,
+  MultiTypeInput,
+  MultiSelectTypeInput,
+  Button,
+  Icon
+} from '@harness/uicore'
+import { map, find } from 'lodash-es'
+import { useStrings } from 'framework/strings'
+import type { ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import {
+  FormMultiTypeDurationField,
+  getDurationValidationSchema
+} from '@common/components/MultiTypeDuration/MultiTypeDuration'
+import { IdentifierSchemaWithOutName, NameSchema } from '@common/utils/Validation'
+import { useVariablesExpression } from '@pipeline/components/PipelineStudio/PiplineHooks/useVariablesExpression'
+import { ConfigureOptions } from '@common/components/ConfigureOptions/ConfigureOptions'
+import MultiTypeFieldSelector from '@common/components/MultiTypeFieldSelector/MultiTypeFieldSelector'
+import { FormMultiTypeConnectorField } from '@connectors/components/ConnectorReferenceField/FormMultiTypeConnectorField'
+import { StepViewType, setFormikRef, StepFormikFowardRef } from '@pipeline/components/AbstractSteps/Step'
+import { useListAwsRegions } from 'services/portal'
+import { useCFCapabilitiesForAws, useCFStatesForAws, useGetIamRolesForAws } from 'services/cd-ng'
+import { Connectors } from '@connectors/constants'
+import { TFMonaco } from '../../Common/Terraform/Editview/TFMonacoEditor'
+import CFRemoteWizard from '../ConnectorStep/CFRemoteWizard'
+import { InlineParameterFile } from './InlineParameterFile'
+import { onDragStart, onDragEnd, onDragLeave, onDragOver, onDrop } from '../DragHelper'
+
+import stepCss from '@pipeline/components/PipelineSteps/Steps/Steps.module.scss'
+import css from '../CloudFormation.module.scss'
+
+interface CloudFormationCreateStackProps {
+  allowableTypes: MultiTypeInputType[]
+  isNewStep: boolean | undefined
+  readonly: boolean | undefined
+  initialValues: any
+  onUpdate: (values: any) => void
+  onChange: (values: any) => void
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
  onChange: (values: unknown) => void
```